bindings/python/tests/global_dlite_state_mod3.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import os
+import pickle
```

Comment:
is pickle needed?

Reply:
..., probably left over from the original file and not needed